### PR TITLE
TASK-US4 Update the UI for the website to be more dynamic and colourful-4

### PR DESCRIPTION
**Pull request by JRB958**
This PR has: 

- compile all the UI work including (change theme, form checking, page titles)

- now the website has a lighter theme with better organization of properties for the ease of view

- possible bug fix needed: at first visit to the dashboard, it doesn't show any properties until visited again after going to the profile page and back. 

related stories: #4 #156

### CODE REVIEW

File: frontend/src/components/createProperty/CreateLocker.js [hunk 1]
```diff
@@ -10,79 +11,181 @@ import {
...
+    } else if (isNaN(formData.purchase_price)){
+      errors.purchase_price = "Purchase price must be a number";
+      isValid = false;
+    } else if (parseFloat(formData.purchase_price) < 0){
+      errors.purchase_price = "Invalid number";
+      isValid = false;
+    } else if (!formData.purchase_price.match(/^(\d*\.{0,1}\d{0,2}$)/)){
+      errors.purchase_price = "Purchase price can only have 2 decimal places";
+      isValid = false;
+    } else if (formData.purchase_price > 20){
```
Comment (NotTh3ta) [inline frontend/src/components/createProperty/CreateLocker.js:78]: This verification is not consistent with the error message

Comment (JRB958) [reply frontend/src/components/createProperty/CreateLocker.js:78]: Agree! a bug task was create to address this issue ( #158) ! Thanks

**Review by NotTh3ta — APPROVED** [review]
Changes to the UI do not impact site functionality.
Comments are descriptive and the commits are atomic.
Form validation needs some work. Some fields should check for the length of the input.
Overall, the ui design is much better.

**Review by MinhDuc1711 — APPROVED** [review]
Merged UI changes to development
Added type check and validation
Improved UI with page titles and additional images
Minor bug fixes still required for type checking

**Pull request merged**